### DISCUSSION TITLE
Validated or RH-Certified?

### DISCUSSION THREAD

**Discussion by erraticx** [containers/ansible-podman-collections#596] [General]
Can this collection be placed into the verified or RH_Certified repo? I would like to be able to use from my PAH.

**Comment by sshnaidm** [comment]
This collection is not officially supported by RH, so I don't think it's possible.
What is "PAH"?

**Reply by erraticx** [reply]
Private Automation Hub. I know this is used pretty heavily with automation controller and creating EEs so assumed it could be added to RH aautomation Hub

**Reply by sshnaidm** [reply]
If it's your own hosted automation hub, it can be added for sure. But it can't be added to RH certified Automation Hub, because it's only for supported collections by RH.
